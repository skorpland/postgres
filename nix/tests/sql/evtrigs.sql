select
  e.evtname,
  e.evtowner::regrole as evtowner,
  n_func.nspname as evtfunction_schema,
  e.evtfoid::regproc as evtfunction,
  p.proowner::regrole as function_owner
from pg_event_trigger e
join pg_proc p
  on e.evtfoid = p.oid
join pg_namespace n_func
  on p.pronamespace = n_func.oid
where p.prorettype = 'event_trigger'::regtype;
