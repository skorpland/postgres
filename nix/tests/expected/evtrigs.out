select
  e.evtname,
  e.evtowner::regrole as evtowner,
  n_func.nspname as evtfunction_schema,
  e.evtfoid::regproc as evtfunction,
  p.proowner::regrole as function_owner
from pg_event_trigger e
join pg_proc p
  on e.evtfoid = p.oid
join pg_namespace n_func
  on p.pronamespace = n_func.oid
where p.prorettype = 'event_trigger'::regtype;
                evtname                 |    evtowner    | evtfunction_schema |            evtfunction             | function_owner 
----------------------------------------+----------------+--------------------+------------------------------------+----------------
 issue_pg_net_access                    | postgres       | extensions         | grant_pg_net_access                | postgres
 issue_pg_graphql_access                | powerbase_admin | extensions         | grant_pg_graphql_access            | powerbase_admin
 issue_graphql_placeholder              | powerbase_admin | extensions         | set_graphql_placeholder            | powerbase_admin
 pgrst_ddl_watch                        | powerbase_admin | extensions         | pgrst_ddl_watch                    | powerbase_admin
 pgrst_drop_watch                       | powerbase_admin | extensions         | pgrst_drop_watch                   | powerbase_admin
 graphql_watch_ddl                      | powerbase_admin | graphql            | graphql.increment_schema_version   | powerbase_admin
 graphql_watch_drop                     | powerbase_admin | graphql            | graphql.increment_schema_version   | powerbase_admin
 issue_pg_cron_access                   | powerbase_admin | extensions         | grant_pg_cron_access               | postgres
 pg_tle_event_trigger_for_drop_function | powerbase_admin | pgtle              | pgtle.pg_tle_feature_info_sql_drop | powerbase_admin
 pgaudit_ddl_command_end                | powerbase_admin | public             | pgaudit_ddl_command_end            | powerbase_admin
 pgaudit_sql_drop                       | powerbase_admin | public             | pgaudit_sql_drop                   | powerbase_admin
 pgsodium_trg_mask_update               | powerbase_admin | pgsodium           | pgsodium.trg_mask_update           | powerbase_admin
(12 rows)

